Think about expanding interop section (#48)
BIDS-numpy/numpy-paper: body.tex

% Ralf: Could you provide some proof of effectiveness, e.g., "This has made it
% possible to create distributed GPU arrays, enabling .... [find a reference to
% work by Peter Entschev]".
% Below an attempt to start on this:
Thanks to these developments users of Dask and NumPy arrays are now capable

> to scale their computation to the GPU using Dasks CuPy integration with

Reviewer: ```suggestion
of scaling their computation to the GPU using dask's CuPy integration with
```

> only minimal code changes.

Author: Honestly, I just realized I have to check that this tool chain is actually has `__array_function__` as a requirement... I assume this is the case though.

Author: Ah, OK, it is spelled out on this page: https://docs.dask.org/en/latest/gpu.html

EDIT: The video cited embedded there could be a citation: https://on-demand-gtc.gputechconf.com/gtcnew/sessionview.php?sessionName=s9198-dask+and+v100s+for+fast%2c+distributed+batch+scoring+of+computer+vision+workloads

However, general adoption has not yet been achieved mainly for compatibility

> concerns.

Reviewer: Could you explain a bit what "compatibility concerns" means?

> Thus, we are actively exploring to address these by revising and extending

Reviewer: ```suggestion
Thus, we are actively addressing these concerns by revising and extending
```

> these protocols with the aim of achieving a situation where this

Reviewer: ```suggestion
these protocols with the aim of wide
```

> interoperability spans the whole scientific Python ecosystem up to and including domain libraries.

Reviewer: ```suggestion
interoperability that spans the whole scientific Python ecosystem up to and including domain libraries.
```

Subclasses of NumPy often exist due to difficulty in constructing richer data
types, such as quantities with physical units \cite{astropy,Goldbaum2018,pint},
geometrical objects \cite{pygeos}, and missing numbers. % pandas: categoricals, IP addresses?
To improve \emph{extensibility}, we are currently overhauling the data type
system to make it easier to implement data types both in Python and C, and to
support these and other applications.

> The widespread adoption of rich, user defined, data types can allow more

Reviewer: ```suggestion
The widespread adoption of rich, user defined data types can allow more
```

concise, feature rich scientific computing.
By making these user implemented data types a fundamental design of NumPy

> these tools will enabled and interact naturally with different libraries.

Reviewer: maybe a bit of clarity on what "enable and interact naturally with different libraries" means.

Author: Yeah, I would think of 1. I expect multiple user dtypes can interact (naturally); 2. libraries such as sklearn (or dask) might "just work" with e.g. physical Units (and Categoricals, which they probably really would like to use e.g. for labels); 3. Things being inside a dtype rather than subclass is a better user experience.
I agree its phrased much too fuzzy, but can't think of an improvement quickly right now, maybe the solution is just to expand it to a few sentences though.

Reviewer 2: That sounds reasonable and I agree that you should expand this into a few sentences or more.
